### PR TITLE
Fixes #12

### LINKED ISSUE #12: Implement parser for Telegram chatfiles in HTML format
Implement a new parser enabling the processing of Telegram export files. Telegram files may be exported in both JSON and HTML format.
This issue targets the parsing of **HTML** files. The corresponding issue for JSON files is #11.
The logic should be aligned with the existing parser classes.

Please find more information about exporting Telegram chatlogs [here](https://telegram.org/blog/export-and-more).

### PR DESCRIPTION
**Pull request by gajrajgchouhan**
- Fixes #12 
- There was an error in json parser of telegram.
- I will try to check for more telegram chat files or this works fine too for the time being.

### CODE REVIEW

File: chatminer/chatparsers.py [hunk 1]
```diff
@@ -237,12 +238,32 @@ def _read_file_into_list(self):
...
+            "author": mess["from"],
+            "datetime": datetime.datetime.fromtimestamp(int(mess["date_unixtime"])),
             "message": mess["text"],
         }
         return parsed_message
 
+class TelegramHtmlParser(Parser):
+    def _read_file_into_list(self):
+        with self._file.open(encoding="utf-8") as f:
+            soup = BeautifulSoup(f, "html.parser")
```
Comment (joweich) [inline chatminer/chatparsers.py:250]: why not use the [native Python HTML parser](https://docs.python.org/3/library/html.parser.html)? BeautifulSoup unnecessarily pollutes the dependencies.

File: chatminer/chatparsers.py [hunk 2]
```diff
@@ -237,12 +238,32 @@ def _read_file_into_list(self):
 
     def _parse_message(self, mess):
         parsed_message = {
-            "datetime": mess["from"],
-            "author": datetime.datetime.fromtimestamp(int(mess["date_unixtime"])),
+            "author": mess["from"],
+            "datetime": datetime.datetime.fromtimestamp(int(mess["date_unixtime"])),
```
Comment (joweich) [inline chatminer/chatparsers.py:242]: Good catch!

File: requirements.txt [hunk 3]
```diff
@@ -1,6 +1,17 @@
...
+packaging==21.3
+pandas==1.5.0
+Pillow==9.2.0
+pyparsing==3.0.9
+python-dateutil==2.8.2
+pytz==2022.4
+seaborn==0.12.0
+six==1.16.0
+soupsieve==2.3.2.post1
+wordcloud==1.8.2.2
```
Comment (joweich) [inline requirements.txt:17]: There are some unused dependencies (e.g. seaborn).

**Review by joweich — CHANGES_REQUESTED** [review]
Left some minor comments, looks great otherwise!

File: chatminer/chatparsers.py (hunk 1 above)
Comment (gajrajgchouhan) [reply chatminer/chatparsers.py:250]: The native one doesn't have the find capability using class, tag, etc, and other types of traversals afaik. 
I have used BeautifulSoup before, and also it uses `html.parser` for traversing anyway (which is the native HTML Parser you were talking about).

File: requirements.txt (hunk 3 above)
Comment (gajrajgchouhan) [reply requirements.txt:17]: Ohh, I just used `pip3 freeze` to generate it.

**Pull request merged**